### ISSUE TITLE
The RISC-V Zig Example is failing on macOS

### ISSUE THREAD

**Issue by umarcor** [im-tomu/fomu-workshop#609]
> https://github.com/im-tomu/fomu-workshop/runs/4842918465?check_suite_focus=true#step:10:6456
```
   ./build.zig:17:29: error: expected type 'std.build.FileSource', found '*const [12:0]u8'
      elf.setLinkerScriptPath("ld/linker.ld");
                              ^
  /usr/local/Cellar/zig/0.9.0/lib/zig/std/build.zig:1391:24: note: std.build.FileSource declared here
  pub const FileSource = union(enum) {
```

/cc @daurnimator

**Comment by daurnimator** [comment]
It appears the code needs to be updated to zig 0.9.0; it was last updated for 0.8.0 (https://github.com/im-tomu/fomu-workshop/pull/520)

**Comment by umarcor** [comment]
Fixed in #610.
